spec/factories/position.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :position, class: Feedbook::Position do
+    name 'CEO'
+    job_description 'Rule them All'
+    association :department, factory: :department
+  end
+end
```

Comment:
Github complains that there is no newline at the end of the file, maybe your editor is not using LF notation.
What do you think about to add a newline at end of the file?